nautobot/core/events/__init__.py

```diff
@@ -12,6 +16,32 @@
 logger = logging.getLogger(__name__)
 
 
+def load_event_brokers(settings):
+    """Process plugins and log errors if they can't be loaded."""
+    if not getattr(settings, "NAUTOBOT_EVENT_BROKERS", None):
+        return
+
+    for publisher_name, publisher in settings.NAUTOBOT_EVENT_BROKERS.items():
+        options = publisher.get("OPTIONS", {})
+        include_topics = None
+        exclude_topics = None
+        if topics := publisher.get("TOPICS"):
+            include_topics = topics.get("INCLUDE")
+            exclude_topics = topics.get("EXCLUDE")
+        options.update({"include_topics": include_topics, "exclude_topics": exclude_topics})
+        try:
+            event_publisher_path = publisher["CLASS"]
+            module_path, class_name = event_publisher_path.rsplit(".", 1)
+            module = importlib.import_module(module_path)
+            event_publisher_class = getattr(module, class_name)
+            event_broker = event_publisher_class(**options)
+            register_event_broker(event_broker)
```

Comment:
More error handling please - malformed event_publisher_path, event_publisher_class not found in the module, etc.